audeer/core/utils.py

```diff
@@ -257,6 +257,52 @@ def git_repo_version(
     return version
 
 
+def is_semantic_version(version: str) -> bool:
+    r"""Check if given string represents a `semantic version`_.
```

Comment:
Do we have to mention that the version string can start with `v`?

Reply:
I removed that sentence from this docstring. It's shown in the eample, but it's also fine with me to introduce it back.

Reply:
Ok. Then let's leave it as it is.